.github/workflows/anonymization-backend-release.yml

```diff
@@ -0,0 +1,125 @@
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      title:
+        description: 'Title for the release'
+        required: true
+        type: string
+
+      description:
+        description: 'Human-readable description of the release'
+        required: true
+        type: string
+
+      ref:
+        description: 'Commit to base the release on (defaults to `HEAD`)'
+        required: false
+        type: string
+
+      tag:
+        description: 'Tag for the release (defaults to `anonymization-backend-<COMMIT>`)'
+        required: false
+        type: string
+
+permissions:
+  contents: write
+
+env:
+  NAME: anonymization-backend
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  ok:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Bazel environment
+        uses: bazel-contrib/setup-bazel@0.9.1
+
```

Comment:
Could you rewrite this to use our own runners and dev container like:

```
    container:
      image: ghcr.io/dfinity/ic-build@sha256:4fd13b47285e783c3a6f35aadd9559d097c0de162a1cf221ead66ab1598d5d45
      options: >-
        -e NODE_NAME --privileged --cgroupns host -v /cache:/cache -v /var/sysimage:/var/sysimage -v /var/tmp:/var/tmp -v /ceph-s3-info:/ceph-s3-info
    runs-on:
      group: zh1
      labels: dind-large
```

That way you don't need to use that setup-bazel action and you benefit from using our bazel-remote cache.